guiEditor/src/diagram/workbench.tsx

```diff
@@ -397,7 +440,28 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
             removeObservers();
         }, PointerEventTypes.POINTERUP);
 
-        scene.onPointerObservable.add(zoomFn, PointerEventTypes.POINTERWHEEL);
+
+        scene.onKeyboardObservable.add((k: KeyboardInfo, e: KeyboardEventTypes) => {
+            switch (k.event.key) {
+                case "q": //select?

```

Comment:
When defined above, you are using a capital "Q" , but here is is lowercase. Is that the same shortcut?